### PR TITLE
No need these deps in apmon

### PR DESCRIPTION
**Pull request by juztas**

### CODE REVIEW

**Comment by juztas** [comment]
@amaltaro can you take a look ?

**Comment by ticoann** [comment]
test this please

File: src/python/WMCore/Services/Dashboard/ProcInfo.py [hunk 1]
```diff
@@ -32,17 +32,14 @@
...
-from past.utils import old_div
 from WMCore.Services.Dashboard.Logger import Logger
 import socket
 import os
 import re
 import time
 
 
-class ProcInfo(object):
+class ProcInfo:
```
Comment (ticoann) [inline src/python/WMCore/Services/Dashboard/ProcInfo.py]: If you want you can still new class inheriting object without importing builtins object. But it is up to you.

**Comment by ticoann** [comment]
Justas, Ideally it would be better to ship all the dependencies on WorkerNode. (in this case future package). But for now this would be fine. Thanks

**Comment by amaltaro** [comment]
Justas, I'm in favor of using the new style class definition, so
`class Logger(object):`

as Seangchan said, there is no need to import from builtins.
If it's using the old style, ok'ish not to update it. But if it has the new style and you make it old, then ... well, not good.

**Comment by amaltaro** [comment]
@juztas if you have time, can you please update the class definition only? Otherwise, Seangchan will fix it in another commit in an hour or two.

**Comment by juztas** [comment]
@amaltaro have you did that already? if not I will change it now. Please, let me know and sorry for late reply

**Comment by amaltaro** [comment]
Nope, otherwise this one would be merged

**Comment by ticoann** [comment]
@juztas, Don't worry about it. I will create additional patch. Thanks,

**Pull request merged**